packages/angular/src/generators/setup-mfe/schema.json

```diff
@@ -45,6 +45,10 @@
       "type": "boolean",
       "default": false,
       "description": "Do not add dependencies to `package.json`."
+    },
+    "e2eProjectName": {
+      "type": "string",
+      "description": "The project name of the associated e2e project for the application."
```

Comment:
Maybe we could also say that's only needed for Cypress e2e projects with a name that doesn't follow the `<app>-e2e` naming convention.

Reply:
yeah good idea